lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb

```diff
@@ -267,7 +267,7 @@ def self.extract_test_results(ftl_service, test_results, gcp_project, history_id
             testCases.each do |testCase|
               name = testCase["testCaseReference"]["name"]
               status = testCase["status"]
-              if status.nil? 
+              if (print_successful_test == true && status.nil?)
```

Comment:
I think we do not need the `== true` and it works if we place it only in this `if status.nil? ` otherwise if the flag is false it will still print fire ones 🔥 😄

Reply:
Goood catch, hahahahaha, Oooops!